lib/razorpay/order.rb

```diff
@@ -25,5 +25,13 @@ def payments(options = {})
       r = self.class.request
       r.request :get, "/orders/#{id}/payments", options
     end
+    
+    def self.edit(id, options = {})
+      request.patch id, options
+    end
+
+    def self.fetchTransferOrder(id)
```

Comment:
nit: snakecase

Reply:
Missing doc for this

Reply:
i have doc and change function in snakecase

Reply:
SDK doc is missing. What you added is a link to the doc. - https://github.com/razorpay/razorpay-ruby/pull/132/commits/33873b01c43c4a8baf9ab46a6836a4f9ed6d7015#diff-4e51636c699f2c237011920308f14d500ee93b09bef25443e5c2720171900954R35

Reply:
\<internal\> https://razorpay.slack.com/archives/C029ZL11PNK/p1649335900167959?thread_ts=1647927334.987909&cid=C029ZL11PNK